### ISSUE TITLE
Unable to compile Sandbox when building from source

### ISSUE THREAD

**Issue by tkuchida** [opensim-org/opensim-core#2231]
Reported on the Forum ([topic 9331](https://simtk.org/plugins/phpBB/viewtopicPhpbb.php?f=91&t=9331)).

**Comment by chrisdembia** [comment]
IMO it is not necessary that we update the sandbox files as the API evolves.

**Comment by tkuchida** [comment]
> IMO it is not necessary that we update the sandbox files as the API evolves.

Agreed. My understanding of the Forum post was that *All Build* also builds the Sandbox, which perhaps it shouldn't.

**Comment by chrisdembia** [comment]
`ALL_BUILD` does not build the Sandbox. However, if you right-click the entire solution `OpenSim` and click `Build`, *all* projects are built, regardless of whether or not they are part of `ALL_BUILD`.

**Comment by idhamari** [comment]
If there is no need to build it probably it is better to execlude it from the CMakeLists.txt file
